### DISCUSSION TITLE
ENG BLEED

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6372] [General]
Hi developers,

seems to be a problem with the ENG BLEED. Using latest dev version. Fault without ECAM message during climb.

![grafik](https://user-images.githubusercontent.com/76531893/146348736-889e81f1-84bc-4fa4-884f-aa0354763980.png)

![grafik](https://user-images.githubusercontent.com/76531893/146348804-71f9dd1c-b887-48f0-9b7b-21ec542ad2f5.png)

greets

**Comment by tracernz** [comment]
cc @BlueberryKing

**Comment by BlueberryKing** [comment]
This should be fixed as of the latest PR (#6359). Make sure to get the current development version! 😄
